neqo-common/src/log.rs

```diff
@@ -46,26 +46,26 @@ macro_rules! qlog {
 }
 #[macro_export]
 macro_rules! qerror {
-    ([$ctx:expr] $($arg:tt)*) => ( ::neqo_common::qlog!(::log::Level::Error, $ctx, $($arg)*); );
+    ([$ctx:expr], $($arg:tt)*) => ( ::neqo_common::qlog!(::log::Level::Error, $ctx, $($arg)*); );
```

Comment:
There is a minimal change here that might have made this easier.

```suggestion
    ([$ctx:expr] $(,)? $($arg:tt)*) => ( ::neqo_common::qlog!(::log::Level::Error, $ctx, $($arg)*); );
```

Reply:
Oh does it make the comma optional ? 
TIL Thanks!

Reply:
I'm going to propagate the changes everywhere then